### PR TITLE
fix for:  error during normal operation

### PR DESCRIPTION
**Pull request by prekucki**

### CODE REVIEW

**Review by mfranciszkiewicz — APPROVED** [review]

**Comment by maaktweluit** [comment]
Please wait for approval to postpone the release before mering this

**Comment by tworec** [comment]
Tihs is **NOT** to be merged before `beta.1` release. It will be a part of the patch release.
Pls do not merge it before `beta.1` tag is created.

**Pull request merged**
